src/app/components/Messages/UserMessage.vue

```diff
@@ -4,47 +4,80 @@
     @mouseleave="hover = false"
     @mouseover="hover = true"
   >
-    <div>
+    <div :style="{ textAlign: isNotice ? 'center' : 'left'}">
       <span :class="{'chat-message-action': isAction}">
         <Username
           :user="message.user"
           :color="usernameColor"
           :is-action="isAction"
+          @focus="hover = true"
+          v-if="!isNotice && (!sameUserAsPrevious || isAction)"
```

Comment:
I am not sure whether that's the correct way to handle multiple messages by the same author; Shouldn't we have a separate `startOfMessagesByOneUser` or `messagesGroup` comp?
The rational is that if a user posts a couple of messages in a row, and is then muted, all of them will collapse into one message, so having a `MessageGroup` component may be a design element that will be necessery either way (unless that collapsing logic will be handled by changing the messages array).
@luxaritas thoughts?

Reply:
A MessageGroup complement seems like the way to go. Gives us more latitude if we want to adjust the styling (or content) of the group as a whole as well